src/indexing/enclave_lfs_access.h

```diff
@@ -0,0 +1,282 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ccf/indexing/lfs_interface.h"
+#include "crypto/entropy.h"
+#include "crypto/hash.h"
+#include "crypto/symmetric_key.h"
+#include "ds/hex.h"
+#include "ds/messaging.h"
+
+#include <optional>
+#include <set>
+#include <unordered_map>
+
+// Uncomment to disable encryption and obfuscation, writing cache content
+// directly unencrypted to host disk
+// #define PLAINTEXT_CACHE
```

Comment:
Can we add another check to make sure this isn't set when built in the enclave?

Reply:
I've added this, so I'm not sure how useful it will be. If we ever want to test the enclave build with this debugging feature, we'll just comment out the lines, and have the same check on PRs that we don't check that in.